tests/assets/js-error-attribute-precedence.html

```diff
@@ -11,11 +11,12 @@
 
     <script>
       newrelic.setCustomAttribute('localKey', 1)
+      setTimeout(function() {
       try {
         throw new Error("My error")
       } catch (e) {
         NREUM.noticeError(e)
-      }
+      }},100)
```

Comment:
Q: Just to double check, is there an edge case of our `noticeError` API not working on tick 0 of the event loop or something?

Reply:
lets validate this

Reply:
it **should** work

Reply:
validated, appears to work as intended